### PR TITLE
suggestions updated

### PR DESCRIPTION
**Pull request by amanueltsfy**

### CODE REVIEW

File: src/components/Wizard.tsx [hunk 1]
```diff
@@ -27,7 +28,7 @@ const Wizard = ({ userJson }: any) => {
     <>
       <WizardHeader counter={counter} />
       <div className="w-full h-3/4 overflow-y-auto py-6 flex items-center">
-        {counter <= 5 ? (
+        {counter <= 6 ? (
```
Comment (jeznag) [inline src/components/Wizard.tsx:31]: instead of hard coding it, you can do

```
counter <= questions.length
```

Comment (jeznag) [reply src/components/Wizard.tsx:31]: @amanueltsfy you haven't changed this. It needs to be changed everywhere you had hardcoded 6

Comment (amanueltsfy) [reply src/components/Wizard.tsx:31]: image

Comment (jeznag) [reply src/components/Wizard.tsx:31]: The source of truth is github. If it doesn't look right in this pull request, it's not mergeable. Perhaps you didn't push your code to the right branch

File: src/components/WizardBody.tsx [hunk 2]
```diff
@@ -45,8 +45,12 @@ const WizardBody = ({
     userJson["breakActivities"] || DefaultJson["breakActivities"]
   );
 
+  const [inputListQ6, setInputListQ6] = useState(
+    userJson["eveningHabits"] || DefaultJson["eveningHabits"]
+  );
+
   useEffect(() => {
-    counter >= 5 && setSettings(downloadData);
+    counter >= 6 && setSettings(downloadData);
```
Comment (jeznag) [inline src/components/WizardBody.tsx:53]: here

File: src/components/WizardBody.tsx [hunk 3]
```diff
@@ -212,6 +237,30 @@ const WizardBody = ({
               />
             </div>
           )}
+          {counter === 6 && (
```
Comment (jeznag) [inline src/components/WizardBody.tsx:240]: here

File: src/components/WizardHeader.tsx [hunk 4]
```diff
@@ -74,6 +74,22 @@ const WizardHeader = ({ counter }: any) => {
...
+        className={`w-2/12 h-1 ${counter >= 5 ? "bg-blue-600" : "bg-gray-400"}`}
+      />
+      <div
+        className={`w-12 h-10 sm:h-11 flex items-center justify-center rounded-full font-medium ${
+          counter >= 6
+            ? "bg-blue-600 text-white"
+            : "bg-transparent text-gray-600 border-2 border-gray-400"
+        }`}
+      >
+        {counter <= 6 ? (
```
Comment (jeznag) [inline src/components/WizardHeader.tsx:87]: here

File: src/components/WizardHeader.tsx [hunk 5]
```diff
@@ -74,6 +74,22 @@ const WizardHeader = ({ counter }: any) => {
           <img className="scale-50" src={checked} alt="checked" />
         )}
       </div>
+      <hr
+        className={`w-2/12 h-1 ${counter >= 5 ? "bg-blue-600" : "bg-gray-400"}`}
+      />
+      <div
+        className={`w-12 h-10 sm:h-11 flex items-center justify-center rounded-full font-medium ${
+          counter >= 6
```
Comment (jeznag) [inline src/components/WizardHeader.tsx:82]: here

**Pull request merged**